src/applications/disability-benefits/2346/Components/OrderPage.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+const OrderPage = () => {
```

Comment:
Same here: https://github.com/department-of-veterans-affairs/vets-website/pull/11679/files#r377926706

Reply:
This file has been deleted.